### PR TITLE
Remove old VictoryTheme files

### PR DESCRIPTION
**Pull request by ghost**
Use centralized styles and remove old VictoryTheme files.
Centralize BaseMap resources and add the disaster-game theme

### CODE REVIEW

File: packages/2018-housing-affordability/src/components/ExploreHousingPolicyImplementation/SimpleCircle.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import { css } from "emotion";
 
-import CivicVictoryTheme from "@hackoregon/component-library/src/VictoryTheme/VictoryThemeIndex";
+import { VictoryTheme } from "@hackoregon/component-library";
```
Comment (jaronheard) [inline packages/2018-housing-affordability/src/components/ExploreHousingPolicyImplementation/SimpleCircle.js]: 🎉🎉

File: packages/component-library/src/StackedAreaChart/StackedAreaChart.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ import {
   getDefaultDataSeriesLabels,
   getDefaultAreaStyle
 } from "../utils/chartHelpers";
-import CivicVictoryTheme from "../VictoryTheme/VictoryThemeIndex";
+import { VictoryTheme } from "../_Themes/index";
```
Comment (jaronheard) [inline packages/component-library/src/StackedAreaChart/StackedAreaChart.js]: Should set all of these components to have this as a default theme, and add a theme prop

**Review by jaronheard — COMMENTED** [review]
Looking good, just a couple of comments for future improvements!

File: packages/component-library/src/StackedAreaChart/StackedAreaChart.js (hunk 2 above)
Comment (Dianna) [reply packages/component-library/src/StackedAreaChart/StackedAreaChart.js]: Great point! Will do

Comment (Dianna) [reply packages/component-library/src/StackedAreaChart/StackedAreaChart.js]: Done 🎨

File: packages/component-library/stories/Scatterplot.story.js [hunk 3]
```diff
@@ -279,6 +292,7 @@ export default () =>
             invertX={invertX}
             invertY={invertY}
             legendComponent={customLegend}
+            theme={(name => themes[name])(theme)}
```
Comment (jaronheard) [inline packages/component-library/stories/Scatterplot.story.js]: I actually didn't know you could do this! Cool!

**Review by jaronheard — APPROVED** [review]
fun-times

**Pull request merged**